### DISCUSSION TITLE
Autoclick run configuration after start time is passed.

### DISCUSSION THREAD

**Discussion by dharmesh-hemaram** [Dhruv-Techapps/acf-docs#364] [Ideas]
@Dhruv-Techapps/developer  btw if i set the start time for example to 14:13:00:00, but it is already 15:34:00, the autofill tool starts the configuration immediately, even the start time was already gone... please fix this

**Comment by dharmesh-hemaram** [comment]
We will think of this scenario. Currently, our extension does not store any information on the page that it completed on a specific page or so. It does not have information like when to stop and when not to.

We will surely

**Comment by dharmesh-hemaram** [comment]
Probably something like end time would help?

**Reply by Scoty2148** [reply]
please re-introduce Schedule Timer that auto clicker start perform action in a specific time
